### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by jadibdev**
### Description
Added styling for the navbar component, styles for the book component, and styles for the form component.

### Screenshot
Screen Shot 2022-07-15 at 12 26 56 PM

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,31 +1,137 @@
...
+  margin-right: 10px;
+}
+
+#update__button {
+  width: 150px;
+  background: #0290ff;
+  color: white;
+  height: 30px;
+  cursor: pointer;
+}
```
Comment (hiromataba) [inline src/index.css]: - [x] Please remove the border on your button to match the provided design 

Your design 👇 
image

Original Design 👇 

image

File: src/index.css [hunk 2]
```diff
@@ -1,31 +1,137 @@
...
+  font-size: 1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.navbar__container {
```
Comment (hiromataba) [inline src/index.css:44]: - [x] Please double check the font and colors even the layout of the original design , like your user icon and also the alignment of your logo and books and categories are not on the same line

Original design 👇 

image

Your design 👇 

image

File: src/index.css [hunk 3]
```diff
@@ -1,31 +1,137 @@
...
+  background: white;
+  margin: 50px;
+  padding: 40px;
+}
+
+.form__container input {
+  width: 150px;
+  height: 20px;
+  margin-right: 10px;
+}
```
Comment (hiromataba) [inline src/index.css]: - [x] Please make you form match the original design 

Original design 👇 

image

Your design 👇 

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @jadibdev     :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @jadibdev, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**